### PR TITLE
Surface npm script to copy webcomponent polyfills and Helix assets in consuming apps

### PR DESCRIPTION
**Pull request by kamalgill**
Add Node.js command-line `copy-helix-assets` script (maps to `bin/copyAssets.js`) to ease the setup of Helix assets and webcomponents polyfills for consuming applications such as the Global Dashboard.

To test the script, perform the following steps:
1. In the helix-ui repo, run `npm install` and `npm link`.  The latter will surface the `copy-helix-assets` as a global CLI command.
2. Switch to the repo root for a Helix-consuming app such as the Global Dashboard.
3. Run `copy-helix-assets`
4. Try running `copy-helix-assets -o public/foo` to test if the Helix assets are copied to the non-default `public/assets` folder.

### CODE REVIEW

**Comment by HelixUI** [comment]
Deploy preview for *helix-ui* processing.

Built with commit 68617b9a19930dd00b603cacd394d201be03c015

https://app.netlify.com/sites/helix-ui/deploys/5be118ea73f2cf111966a260

File: bin/copyHelixAssets.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
+/* eslint-disable no-console */
+/*
+ * @fileOverview Script that copies static assets such as Helix UI CSS and JS from
+ *   node_modules to public/assets folder
+ *
+ *   NOTE: This script require Node.js 8.5 or later
+ *
+ */
+const chalk = require('chalk');
+const fs = require('fs');
```
Comment (CITguy) [inline bin/copyHelixAssets.js]:
```suggestion
const fs = require('fs-extra');
```

We have [`fs-extra`](https://www.npmjs.com/package/fs-extra) at our disposal that has the same api as `fs` but with some nice convenience functions, too.

**Comment by CITguy** [comment]
In regards to updates to `package.json`:

* We'll have to add any 3rd party npm modules to the "dependencies"
* We'll probably want to configure the ["engines" configuration](https://docs.npmjs.com/files/package.json#engines).
  * I'm guessing it would likely be `>=8.0.0`.

TBH, I've got a gut instinct that this functionality may venture into areas that are beyond the original intent of HelixUI. 

Let's chat in the morning.

File: bin/copyHelixAssets.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            'bundles/webcomponents-*.js' ],
+        // Avoid minifying for now to safely init dynamically loaded webcomponent dependencies
+        minify: false,
+    },
+};
+
+// Ensure /public/assets and /public/assets/bundles folders exist prior to copying assets
+TARGET_FOLDERS.forEach(folder => {
+    if (!fs.existsSync(folder)) {
+        fs.mkdirSync(folder);
```
[the remark this answers is not in the document]
Comment (kamalgill) [reply bin/copyHelixAssets.js]: 👍

File: bin/copyHelixAssets.js (hunk 1 above)
Comment (kamalgill) [reply bin/copyHelixAssets.js]: 👍

**Comment by kamalgill** [comment]
Replaced by https://github.com/HelixDesignSystem/helix-cli/pull/1

**Pull request closed without merging**